docs/shared/common_prereqs/common_prereqs.adoc

```diff
@@ -0,0 +1,181 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2020, 2021 Oracle and/or its affiliates.
```

Comment:
Because this is a new file with this PR, just 2021 for the copyright (even thought some of the content existed previously, in other files.)